.github/workflows/continuous-integration.yml

```diff
@@ -74,7 +74,7 @@ jobs:
     runs-on: ubuntu-22.04
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9, '3.10']
+        python-version: ['3.8', '3.9', '3.10']
```

Comment:
Do they all need to be quoted?

Reply:
Not really, I wasn't sure about the CI. I'll revert these changes